docs/Installation/InstallationOnAppleSilicon.md

```diff
@@ -0,0 +1,170 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+## Installing SpECTRE on Apple Silicon {#installation_on_apple_silicon}
+
+The following instructions show how to install SpECTRE and its dependencies
+on Apple Silicon Macs. Apple Silicon is an arm64 architecture, in contrast
+to the x86-64 architecture that SpECTRE usually targets. These instructions
+will result in an Apple Silicon native build of SpECTRE.
+
+\note Floating-point exception trapping and python bindings are not currently
+supported on Apple Silicon.
+
+### 0. Install the xcode command-line tools.
+
+Install the xcode command-line tools, which include the clang compiler, etc.
+
+~~~
+xcode-select --install
+~~~
+
+### 1. Make a directory to install prerequisites
+
+First, make a directory to hold some prerequisites that spectre depends on.
+Name this directory whatever you like, e.g. `$HOME/apps`.
+~~~
+export SPECTRE_DEPS_ROOT=$HOME/apps
+mkdir $SPECTRE_DEPS_ROOT
```

Comment:
the dollar signs are being rendered as `\f$`

Reply:
This is fixed by using backquotes instead of tildes to bracket code blocks